_topic_map.yml

```diff
@@ -1928,11 +1928,9 @@ Topics:
     File: planning-migration-3-to-4
   - Name: About the Migration Toolkit for Containers
     File: migrating-application-workloads-3-4
-  - Name: Deploying the Migration Toolkit for Containers
+  - Name: Installing and upgrading the Migration Toolkit for Containers
     File: deploying-cam-3-4
-  - Name: Upgrading the Migration Toolkit for Containers
```

Comment:
Why did you move the upgrade section to install? From a modular documentation standpoint, installing and upgrading are different user stories and **should** be in separate assemblies.

Reply:
Upgrade and installation are closely related. The upgrade assembly did not fit well into the upper structure. I checked with Vikram and he said that when he looks for installation procedures, he expects to find upgrade procedures in the same section. It's an interesting topic for discussion. While upgrade might be separate from installation from a mod docs POV, in the user's mind, they are related topics.